tests/e2e/run-e2e-tests.sh

```diff
@@ -0,0 +1,159 @@
+#!/usr/bin/env bash
+# shellcheck disable=SC1091
+
+# Run a full suite of e2e tests using the code and against the cluster defined
+# in the calling environment via the rox-ci-image / stackrox-test container.
+
+set -euo pipefail
+
+ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")"/../.. && pwd)"
+source "$ROOT/scripts/lib.sh"
+source "$ROOT/scripts/ci/lib.sh"
+source "$ROOT/scripts/ci/gcp.sh"
+
+usage() { 
+    cat <<_EOH_
+Usage: $0 [-m <tag>] [-o k8s|openshift] [-y] [<e2e flavor: one of qa|e2e|ui|upgrade, defaults to qa>]
+  -m - override 'make tag' for the version to install.
+  -o - choose the cluster variety. defaults to k8s.
+  -y - run without prompts.
+_EOH_
+    exit 1
+}
+
+if [[ ! -f "/i-am-rox-ci-image" ]]; then
+    kubeconfig="${KUBECONFIG:-${HOME}/.kube/config}"
+    docker run \
+      -v "$ROOT:$ROOT:z" \
+      -w "$ROOT" \
+      -e "KUBECONFIG=${kubeconfig}" \
+      -v "${kubeconfig}:${kubeconfig}:z" \
+      -e VAULT_TOKEN \
+      --platform linux/amd64 \
+      --rm -it \
+      --entrypoint="$0" \
+      quay.io/stackrox-io/apollo-ci:stackrox-test-0.3.52-3-g49dc4af1cd "$@"
```

Comment:
This will eventually use `/BUILD_IMAGE_VERSION`, right?

Reply:
Sure. I was going to just make it a released version when the rox-ci-image PR lands but s/-build-/-test-/ on BUILD_VERSION makes sense.

Reply:
Actually, I'll need to wait until the next BUILD_IMAGE_VERSION bump as the rox-ci-image required for this is later.